content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -11,10 +11,33 @@ The rules engine can radically boost your productivity in the PIM by automating
 
 The rules are defined in a YML file that you can import in the PIM, using a dedicated job profile.   
 
-A rule is defined by a code (required) and you can name it with a label as well. This label can have a value per locale.
-
 This feature is very, like, really powerful, and the use cases are countless. In order to guide you in the rules engine usage, we wrote this article where you will find out what you can use it for and how it works.
 
+# Name your rules
+
+A rule is defined by a code (required) and you can name it with a label as well. This label can have a value per locale.
+Here is the expected format:
```

Comment:
```suggestion
The labels settings must be added at the end of your rule, after the actions part. 
For instance, if you want to add a label for your en_US and fr_FR locales, here is the expected format:
```

Reply:
I guess the label settings can be put before or after, it doesn't matter (I just tested to put it first, before the priority and it seems to work, but @nmarniesse do you confirm?)

Reply:
Yes there is no order for the keys `priority`, `conditions`, `actions` or `labels`. You can put them in the order you want.  
In yaml the order is important only for lists (when you have a `-` in front of the line)

Reply:
Thanks @nmarniesse , I've edited my suggestion @morganen :)